README.md

```diff
@@ -8,8 +8,7 @@
 [![GitHub contributors](https://img.shields.io/github/contributors/sendgrid/sendgrid-csharp.svg)](https://github.com/sendgrid/sendgrid-csharp/graphs/contributors)
 [![Open Source Helpers](https://www.codetriage.com/sendgrid/sendgrid-csharp/badges/users.svg)](https://www.codetriage.com/sendgrid/sendgrid-csharp)
 
-<a name="announcements"></a>
-# Announcements
```

Comment:
nit: put these back, unnecessary change.

Reply:
I think this is fine to merge header and anchor into single line.

See https://gist.github.com/asabaylus/3071099#gistcomment-2665317 for reference